providers/fab/src/airflow/providers/fab/auth_manager/security_manager/override.py

```diff
@@ -233,6 +233,7 @@ class FabAirflowSecurityManagerOverride(AirflowSecurityManagerV2):
         (permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG_CODE),
         (permissions.ACTION_CAN_READ, permissions.RESOURCE_DAG_RUN),
         (permissions.ACTION_CAN_READ, RESOURCE_ASSET),
+        (permissions.ACTION_CAN_READ, RESOURCE_ASSET_ALIAS),
```

Comment:
Sorry I missed that during the review but other permissions than READ need to be added to the OP role. @rawwar could you please create a PR for that?

Reply:
@vincbeck , we only GET asset alias. No other verbs on asset aliad. Should i still add others?